plans/placebo/main.go

```diff
@@ -14,8 +16,27 @@ func run(runenv *runtime.RunEnv) error {
 		panic("test case sequence number not set")
 	}
 
-	if runenv.TestCaseSeq != 0 {
+	switch runenv.TestCaseSeq {
+	case 0:
+		return nil
+	case 2:
+		// expose prometheus endpoint
+		listener := runenv.MustExportPrometheus()
+		defer listener.Close()
+
+		// create context for cancelation
+		ctx, cancel := context.WithCancel(context.Background())
+		defer cancel()
+
+		// snapshot metrics every second and save them into "metrics" directory
+		err := runenv.HTTPPeriodicSnapshots(ctx, listener.Addr().String(), time.Second, "metrics")
```

Comment:
I'd imagine that most users will want these snapshots to be taken as long as the process is running (i.e. the lifetime of `context.Background()`), so in most cases folks can just pass in `context.Background()` here, but I like that the control is on the user, i.e. if we want to take snapshots during a particular stage of the test only, we can do so by using a cancellable context 👍